### PR TITLE
Onboards typescript for a single file

### PR DESCRIPTION
**Pull request by godwinpang**

### CODE REVIEW

**Comment by thaigillespie** [comment]
The staging deploy broke so I'll wait until you fix it? Or is this still reviewable?

**Comment by godwinpang** [comment]
u can still review it

File: src/react-app-env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```
Comment (thaigillespie) [inline src/react-app-env.d.ts:1]: What is this file supposed to be?

Comment (godwinpang) [reply src/react-app-env.d.ts:1]: autogenerated whenever i run npm run start - something from cra

**Comment by godwinpang** [comment]
this is goood right

**Comment by godwinpang** [comment]
nvm - pushed a little more stuff to enable hotreloading

**Pull request merged**